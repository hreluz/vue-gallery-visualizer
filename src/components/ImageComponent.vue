<template>
  <img src="/assets/image.jpg" class="img-fluid" alt="">
</template>
<script lang="ts">
import { defineComponent } from 'vue';
export default defineComponent({
  name: 'ImageComponent',
  components: {
  }
});
</script>