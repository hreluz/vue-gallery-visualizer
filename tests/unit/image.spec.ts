import ImageComponent from '@/components/ImageComponent.vue';
import { shallowMount } from '@vue/test-utils'

describe('Image.vue', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(ImageComponent);
    expect(wrapper.html()).toMatchSnapshot();
  })
})