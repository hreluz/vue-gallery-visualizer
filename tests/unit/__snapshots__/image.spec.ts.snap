// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image.vue should match the snapshot 1`] = `<img src="/assets/image.jpg" class="img-fluid" alt="">`;
